packages/connect-node/src/node-readme.spec.ts

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/* eslint-disable @typescript-eslint/unbound-method */
```

Comment:
It looks like we might be destructuring the router in examples, which means that users are likely to also run into this lint rule. 

We can change the signatures:

```ts
  service: <T extends DescService>(
    service: T,
    implementation: Partial<ServiceImpl<T>>,
    options?: Partial<UniversalHandlerOptions>,
  ) => this;
  rpc: <M extends DescMethod>(
    method: M,
    impl: MethodImpl<M>,
    options?: Partial<UniversalHandlerOptions>,
  ) => this;
```

This makes them function properties instead of methods.

I don't see how this change could cause any problems, but it solves the linter issue for us and for users. WDYT?